modules/api/src/main/java/org/apache/ignite/compute/ComputeTask.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.compute;
+
+import java.util.Collection;
+
+/**
+ * Compute task interface defines a task that can be executed on the cluster. Compute task
+ * handles splitting business logic into multiple jobs, receiving
+ * results from individual jobs executing on remote nodes, and reducing
+ * (aggregating) received jobs' results into final compute task result.
```

Comment:
In Ignite 2 we had a ComputeTask and ComputeJob.

What does the term 'job' means here? 
Let's clarify here if the 'job' is also represented with an object of ComputeTask kind.

Reply:
I've updated the javadoc, please, have a look